lib/foreman/renderer/scope/macros/base.rb

```diff
@@ -108,6 +108,14 @@ def host_status(host, name)
             host.get_status(klass)
           end
 
+          def load_users(includes: nil, preload: nil)
+            load_resource(klass: User, search: '', permission: :view_users, includes: includes, preload: preload)
+          end
+
+          def user_auth_source(user)
+            user.auth_source.name
```

Comment:
I'd prefer to be extra defensive here, doing something like `user.auth_source&.name` would be better